data/arabic_letters.json

```diff
@@ -0,0 +1,118 @@
+[
```

Comment:
I like the usage of JSON; this will help if we decide to add any more fields to the card class.